### ISSUE TITLE
Interpolation of value

### ISSUE THREAD

**Issue by Ser-Gen** [simonsmith/postcss-property-lookup#22]
Some lookupped values may be unitless.
But others must use units:
```sass
/* input */

.body {
	line-height: 1.5;
	font-size: @line-height;
}
```
```css
/* output */

.body {
	line-height: 1.5;
	font-size: 1.5;
}
```

I need to get:
```css
/* output */

.body {
	line-height: 1.5;
	font-size: 1.5em;
}
```

For example, we can use this syntax `font-size: @(line-height)em;`

**Comment by jedmao** [comment]
We can't just guess a unit though, so what are you suggesting? What's wrong with `font-size: @(line-height)em`?

**Comment by Ser-Gen** [comment]
Nothing wrong, but it does not works right now :)
I just want to discuss a better way to fix it.

**Comment by simonsmith** [comment]
Fixed in #23
